### ISSUE TITLE
Menu buttons are missing a label

### ISSUE THREAD

**Issue by VirginiaBalseiro** [dokieli/dokieli#508]
All interactive elements should have a role, name and value. The buttons on the main menu don’t have a name (label). A screen reader will read ‘link’ or ‘button’ but doesn’t say what happens when clicking on it.

See: https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

**Comment by VirginiaBalseiro** [comment]
The `<button>` element already has an implicit role of "button," and its text content (`<span>Edit</span>`) provides an accessible name, so screen readers will announce it as “Edit button.” The title is optional and not relied on (but still present in this case). Issues only arise if the button were icon-only or if the icon’s meaning conflicts with the text, in that case, an aria-label would be needed (but that is not the case here). 

The only button with an issue is the "Sign out" button which is an icon-only button with an aria-label, but the label was not useful text, which is now fixed.
